assets/scss/global.scss

```diff
@@ -1,5 +1,14 @@
 $contentWidth: 1440px;
 $mobileBreakpoint: '<lg';
+$col-fixed-width-330: 330px;
```

Comment:
moved these to global and made them specific.. and added it to a class for ease of use